.github/config/regression-tests.yml

```diff
@@ -0,0 +1,805 @@
+          "tests": {
+            "solidity": "test"
+          }
+        },
+        "solidity": {
+          "version": "0.8.24",
+          "remappings": [
+            "ds-test/=lib/forge-std/lib/ds-test/src/",
+            "forge-gas-snapshot/=lib/forge-gas-snapshot/src/",
+            "forge-std/=lib/forge-std/src/",
+            "openzeppelin-contracts/=lib/openzeppelin-contracts/contracts/",
+            "permit2/=lib/permit2/",
+            "solmate/=lib/solmate/",
+            "solarray/=lib/solarray/src/"
+          ],
+          "settings": {
+            "optimizer": {
+              "runs": 1000000
+            }
+          }
+        },
+        "solidityTest": {
+          "ffi": true,
+          "fsPermissions": {
+            "readWrite": [
+              ".forge-snapshots/"
+            ]
+          }
+        }
+      };
+    ignore: |
+      */integration/*
+
+      # unknown cheatcode with selector 0x7fb5297f
+      test/script/DeployDutch.t.sol
+      test/script/DeployExclusiveDutch.t.sol
+      test/script/DeployPriorityOrderReactor.t.sol
+
+      # unknown cheatcode with selector 0x3cad9d7b
+      test/base/EthOutput.t.sol
+      test/validation-contracts/ExclusiveFillerValidation.t.sol
+      test/lib/NonLinearDutchDecayLib.t.sol
+      test/base/ProtocolFees.t.sol
+      test/fill-macros/DirectTakerFillMacro.t.sol
+      test/executors/SwapRouter02Executor.t.sol
+
+      # unknown cheatcode with selector 0xdd9fca12
+      test/base/BaseReactor.t.sol
+      test/reactors/PriorityOrderReactor.t.sol
+      test/reactors/V3DutchOrderReactor.t.sol
+      test/reactors/V2DutchOrderReactor.t.sol
+      test/reactors/LimitOrderReactor.t.sol
+      test/reactors/BaseDutchOrderReactor.t.sol
+      test/reactors/ExclusiveDutchOrderReactor.t.sol
+      test/reactors/DutchOrderReactor.t.sol
+    ref: 4013dfa4bc53b823b406b035a9b5eb579607eb99
+  Vectorized/solady:
+    forge-version: v0.3.0
+    hardhat-config: |
+      export default {
+        "paths": {
+          "sources": "src",
+          "tests": {
+            "solidity": "test"
+          }
+        },
+        "solidity": {
+          "version": "0.8.28",
+          "remappings": [
+            "forge-std=test/utils/forge-std/"
+          ],
+          "settings": {
+            "optimizer": {
+              "enabled": true,
+              "runs": 1000
+            },
+            "evmVersion": "cancun"
+          }
+        },
+        "solidityTest": {
+          "blockGasLimit": BigInt(100000000),
+          "fsPermissions": {
+            "read": [
+              "./test/data"
+            ]
+          },
+          "fuzz": {
+            "runs": 256
+          },
+          "invariant": {
+            "depth": 15,
+            "runs": 10
+          }
+        }
+      };
+    ignore: |
+      */*7702*
+      */*Transient*
+      */ext/ithaca/*
+      */ext/zksync/*
+
+      # testP256Normalized(uint256,bytes32): unknown cheatcode with selector 0xc453949e
```

Comment:
`publicKeyP256(uint256)` cheatcode which is not yet supported

Reply:
```suggestion
      # testP256Normalized(uint256,bytes32): unknown cheatcode with selector 0xc453949e (publicKeyP256(uint256))
```